src/applications/vaos/tests/newAppointmentFlow.unit.spec.js

```diff
@@ -342,39 +342,6 @@ describe('VAOS newAppointmentFlow', () => {
 
         resetFetch();
       });
-      it('should be requestDateTime page if past appointments request errors', async () => {
```

Comment:
No longer performing the past appointment history fetch here so there shouldn't be any failures to test for.